Blackjack/Game/BlackjackGame.h

```diff
@@ -0,0 +1,45 @@
+//
+//  BlackjackGame.h
+//  Blackjack
+//
+//  Created by Alex on 28/09/2022.
+//
+
+#import <UIKit/UIKit.h>
+#import "Deck.h"
+#import "Player.h"
+
+// MARK: - BlackjackGame Delegate
+
+@protocol BlackjackGameDelegate
+-(void)updateUI;
+-(void)betsOver;
+@end
+
+// MARK: - Game Protocol
+
+@protocol Game
+@property (nonatomic, strong, readonly) NSMutableArray * _Nullable players;
+-(void)startGame:(NSInteger) numberOfPlayers;
+-(void)gameOver;
+@end
+
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface BlackjackGame : NSObject<Game>
+@property (nonatomic, strong, readonly) Player *currentPlayer;
```

Comment:
space between `@interface` and `@property`

Reply:
Done